pentaho-gwt-widgets/src/org/pentaho/gwt/widgets/themes/public/themes/sapphire/globalSapphire.css

```diff
@@ -4311,13 +4311,14 @@ div.prompt-panel .pentaho-toggle-button-container > DIV {
 .IE #preview,
 #selectAll,
 #tempPreviewButton,
-#preview {
+#preview{
   padding: 4px;
   -webkit-box-shadow: none;
   -moz-box-shadow: none;
   box-shadow: none;
-  background: #ffffff;
-  color: #005eaa;
+  background: #f5f6f8;

```

Comment:
@mdamour1976 @wseyler 
These css properties were introduced in #420 according to PJ's [suggestion](http://jira.pentaho.com/browse/BACKLOG-11588) to restore the look of these buttons as links.
I assumed however that since the data source wizard dialog has a white background in the **Sapphire** theme, these links should have a white background as well.
My question is, are these the correct values for **background** and **color** ?
Also, is **IE6-9** still a concern?